lightning/src/routing/pickhardt_router/mod.rs

```diff
@@ -0,0 +1,4 @@
+//! Routing implementation for large payments.
+
+pub mod pickhardt_router;
```

Comment:
We can give credit in the comments of course, but I agree that it would be too much work expected from LDK users to get into the details of Rene's papers.

I suggest using one of these two names: LargePaymentRouter (to express to LDK users that the router should be used for large payment amounts, and DefaultRouter should still be used for smaller amounts for now), or MinCostFlowRouter that suggests that this implementation finds the min cost flow in the cost graph versus shortest path Dijkstra that all other lightning routing implementations currently use.

Do you have any preference?

Reply:
I'd prefer `MinCostFlowRouter` as conceivably there could be another router which handles large payments well.